[ET-1980] - Fix My Tickets Update Button When Only Opt-Out Form Shows (#2890)
the-events-calendar/event-tickets: readme.txt

* Fix - Resolved an issue where Order Status was not populated when exporting the Attendee List using Tickets Commerce. [ET-1883]
* Tweak - Update default footer text of Tickets Emails to include link to website. [ET-1971]
* Fix - Update usage of `method_exists()` to comply with PHP 8.1 standards. [ET-1759]

> * Fix - Update button will now show when the opt-out checkbox shows ont heMy Tickets page. [ET-1980]

Reviewer: ```suggestion
* Fix - Update button will now show when the opt-out checkbox shows on the My Tickets page. [ET-1980]
```

* Fix - Resolved an issue where Order Status was not populated when exporting the Attendee List using Tickets Commerce. [ET-1883]
* Tweak - Update default footer text of Tickets Emails to include link to website. [ET-1971]
* Fix - Update usage of `method_exists()` to comply with PHP 8.1 standards. [ET-1759]

> * Fix - Update button will now show when the opt-out checkbox shows on the My Tickets page. [ET-1980]

Reviewer 2: does the new filter `tribe_tickets_plus_hide_attendees_list_optout` need to appear also in the readme?

Reviewer 2: Great, thanks for clarification @codingmusician !
